### ISSUE TITLE
split identifier into standard and local

### ISSUE THREAD

**Issue by malantonio** [LafayetteCollegeLibraries/spot#180]
this was totally an oversight on my part. we need to expand Publication to have `local_identifier` and `standard_identifier` properties (in place of `identifier`)

### `:standard_identifier`
- isbn
- issn
- handles

### `:local_identifier`
- legacy islandora ids
- internal local helpers (see `lafayette_magazine`)
- a copy of the object id (with `noid:` prefix)

both should point to `dct:identifier` as a predicate.

**Comment by malantonio** [comment]
looks like there's a problem with this:

```ruby
class Publication < ActiveFedora::Base
  # ...
  property :standard_identifier, predicate: ::RDF::Vocab::DC.identifier
  property :local_identifier, predicate: ::RDF::Vocab::DC.identifier
end

pub = Publication.new(standard_identifier: ['hdl:0000/1111'], local_identifier: ['lafayette:rulz'])
pub.local_identifier
# => ['lafayette:rulz']
pub.standard_identifier
# => ['lafayette:rulz']
```

i guess ActiveFedora maps the value to the predicate and assigns the property to read from that predicate -- since `local_identifier` was defined last, it becomes the writable property + `standard_identifier` reads from the predicate.

a way around this is to store all of the identifiers under `:identifier` and use the ui/solr to differentiate between local + standard ids.

**Comment by malantonio** [comment]
closed with #183
